From 7053d8176ce502aabe39bf167f01354654bde5df Mon Sep 17 00:00:00 2001
From: Denil Vira <denil@cumulusnetworks.com>
Date: Thu, 23 Jul 2015 13:52:18 +0000
Subject: bgpd: Fix useless call in bgpd/bgp_mplsvpn.c

Coverity scan ID : 1302528. Calling "decode_label(pnt)" is only useful for its return value, which is
ignored. Removed the call.

Signed-off-by: Denil Vira <denil@cumulusnetworks.com>
---
diff --git a/bgpd/bgp_mplsvpn.c b/bgpd/bgp_mplsvpn.c
index 3d2dade..cf4ab6f 100644
--- a/bgpd/bgp_mplsvpn.c
+++ b/bgpd/bgp_mplsvpn.c
@@ -116,9 +116,6 @@ bgp_nlri_parse_vpnv4 (struct peer *peer, struct attr *attr,
 	  zlog_err ("prefix length is less than 88: %d", prefixlen);
 	  return -1;
 	}
-	
-      /* XXX: Not doing anything with the label */
-      decode_label (pnt);
 
       /* Copyr label to prefix. */
       tagpnt = pnt;;
--
cgit v0.9.0.2
